core/src/main/scala/io/finch/Endpoints.scala

```diff
@@ -403,8 +404,9 @@ trait Endpoints {
   val binaryBodyOption: Endpoint[Option[Array[Byte]]] =
     matches(items.BodyItem)(!_.isChunked)(req =>
       req.contentLength match {
-        case Some(n) if n > 0 => Some(Buf.ByteArray.Shared.extract(req.content))
-        case _ => None
+        case Some(0) => Some(Array.empty)
```

Comment:
Sure. You mean I should add a
```
private val emptyBinaryBodyOption = Some(Array.empty[Byte])
```
above and just use that in the `Some(0)` case, right?